lib/game.rb

```diff
@@ -0,0 +1,82 @@
+require_relative 'board.rb'
+require_relative 'players/human.rb'
+require_relative 'players/computer.rb'
+#require_relative 'player.rb'

```

Comment:
Require statements should not be necessary in class files; the environment.rb should take care of these.